core/quivr_core/llm/llm_endpoint.py

```diff
@@ -70,6 +70,7 @@ def from_config(cls, config: LLMEndpointConfig = LLMEndpointConfig()):
                     else None,
                     azure_endpoint=azure_endpoint,
                     max_tokens=config.max_output_tokens,
+                    temperature=config.temperature,
                 )
             elif config.supplier == DefaultModelSuppliers.ANTHROPIC:
                 _llm = ChatAnthropic(
```

Comment:
linter is raising arguments missing

Reply:
It is an inconsistency on LangChain side, it has always been there